### PR TITLE
Update from upstream

### PR DESCRIPTION
**Pull request by lietu**
- Trivy security scanning
 - `poetry1.6.5` -> `poetry1.1.6`
 - Cleaning up Poetry install to not support 2.7, 3.5, 3.6, 3.7, 3.8, etc.
 - Fully activating Python 3.9 by also replacing `python3`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Builds your Docker images automatically, like magic. Good for handling common base images for all your projects, apps,
 whatever.
 
+This repository builds images to [hub.docker.com/u/lietu](https://hub.docker.com/u/lietu)
```
Comment (lietu) [inline README.md:6]: This should be updated to `digitallivinginternational` e.g. after merged

File: python-base/ubuntu20.04-python3.9/docker/scripts/install_poetry.sh [hunk 2]
```diff
@@ -13,3 +13,13 @@ su "${USER}" -c "python /tmp/get-poetry.py --version ${POETRY_VERSION}"
 
 chmod +x "${POETRY_HOME}"/bin/*
 bash /src/docker/scripts/configure_poetry.sh
+
+# Remove useless old vendor packages
+cd /usr/local/poetry/lib/poetry/_vendor/
```
Comment (lietu) [inline python-base/ubuntu20.04-python3.9/docker/scripts/install_poetry.sh:18]: Poetry bundles `2.7`, `3.5`, `3.6`, `3.7`, `3.8`, and `3.9` vendored packages by default and they can take a fair bit of space.

File: python-base/ubuntu20.04-python3.9/docker/scripts/install_python.sh [hunk 3]
```diff
@@ -5,11 +5,16 @@ set -exuo pipefail
...
 # Install python
 apt-get install -y --no-install-recommends \
-  "python${APT_PYTHON_VERSION}" \
+  "python${PYTHON_VERSION}" \
   python3-distutils
 
 # Set python active
-update-alternatives --install /usr/bin/python python /usr/bin/python3.9 10
+update-alternatives --install /usr/bin/python python "/usr/bin/python${PYTHON_VERSION}" 10
+update-alternatives --install /usr/bin/python3 python3 "/usr/bin/python${PYTHON_VERSION}" 10
```
Comment (lietu) [inline python-base/ubuntu20.04-python3.9/docker/scripts/install_python.sh:13]: Quite a lot of things, including Poetry, actually default to `/usr/bin/python3` so this was pretty important

**Review by joakimnordling — APPROVED** [review]
LGTM

**Review by fbjorn — APPROVED** [review]

**Pull request merged**